app/models/UserStore.ts

```diff
@@ -169,9 +177,9 @@ export const UserStoreModel = types
         lastMessageAt: ev.created_at,
       })
     },
-    async updateChannels(pool: NostrPool) {
+    updateChannels: flow(function* (pool: NostrPool) {
```

Comment:
again, please explain why flow vs async/await is beneficial.   there is only one yield statement so i think there is no difference

Reply:
yup, in our case, there is no difference  between async/await vs flow, I use it just for prevent unexpected behavior in future, it is recommended by MST docs anyway